### PR TITLE
Translation pages and menu changes

### PR DESCRIPTION
**Pull request by alexanbj**
This started as a branch for the new pages for translations, but quickly railroaded into a menu rewrite and a more generic breadcrumb components.

Event though  the translations pages still aren't finished, lets get this merged so we get the new menus in before the next user test. The translation pages are behind a feature flag, so so they won't be visible in staging/production.

This pull request resolves GlobalDigitalLibraryio/issues#156, it also has partial implementations of GlobalDigitalLibraryio/issues#149 and GlobalDigitalLibraryio/issues#150

### CODE REVIEW

File: components/Sidemenu/Sidemenu.js [hunk 1]
```diff
@@ -195,38 +198,50 @@ class Sidebar extends React.Component<Props, State> {
...
               </Link>
             ))}
             <Link passHref route="new" params={{ lang: language.code }}>
-              <MenuItem style={{ marginTop: '3px' }}>
+              <MenuItem thickBorder>
                 <Trans>New arrivals</Trans>
               </MenuItem>
             </Link>
-          </Bar>
+            <Link passHref route="about">
```
Comment (kstigen) [inline components/Sidemenu/Sidemenu.js]: Hmm...denne bør vel egentlig gå til `https://home.digitallibrary.io/about/`

File: lib/auth/index.js [hunk 2]
```diff
@@ -19,14 +19,14 @@ const getAuth = options =>
     responseType: 'token id_token',
     scope: 'openid profile',
     redirectUri: `${getBaseUrl()}/auth/signed-in`,
```
Comment (kstigen) [inline lib/auth/index.js]: Ser at det er konfigurert opp feil i auth0, og login i test fungerer ikke. Auth0 forventer `https://www.test.digitallibrary.io/auth/signed-in` ikke `https://test.digitallibrary.io/auth/signed-in`. Hør evt. med @gunnarvelle hva dere bør gjøre.

File: components/Breadcrumb.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+const Nav = styled.nav`
+  display: flex;
+  align-items: stretch;
+  flex: 1;
+  overflow: hidden;
+  overflow-x: auto;
+  white-space: nowrap;
+`;
+
+const Ol = styled.ol`
```
Comment (kstigen) [inline components/Breadcrumb.js]: Må dette navnet være `0l`? Forstår ikke hva det brukes til...mulig å gi det et bedre navn?

File: components/Menu/MenuItem.js [hunk 4]
```diff
@@ -6,21 +6,30 @@
...
   }
-  transition: background-color 0.2s ease;
+
+  &:active {
+    color: ${theme.colors.dark};
+  }
+
+  ${p =>
+    p.thinBorder &&
+    `border-bottom: 1px solid ${theme.colors.grayLight};`} ${p =>
```
Comment (kstigen) [inline components/Menu/MenuItem.js]: Er det kun mulig å putte farger i theme? Gir det mening å definere f.eks borders der også?

File: pages/user/_translations.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                </Box>
+                <Box>
+                  <H4>{book.title}</H4>
+                  <P color={theme.colors.grayDark}>
+                    <Trans>from {book.publisher.name}</Trans>
+                  </P>
+                  <Box>
+                    {book.language.name}{' '}
+                    <MdArrowForward color={theme.colors.oranges.orange} />{' '}
+                    <strong>Nepali</strong>
```
Comment (kstigen) [inline pages/user/_translations.js]: Hø? Hardkodet `Nepali`?

File: lib/auth/index.js (hunk 2 above)
Comment (alexanbj) [reply lib/auth/index.js]: Dette må fikses utenfor frontenden.

File: components/Breadcrumb.js (hunk 3 above)
Comment (alexanbj) [reply components/Breadcrumb.js]: Det er en ordered list, Ol :)

**Pull request merged**